resources/user-agents/browsers/opera-mobile/opera-mobile-10-00-to-10-70.json

```diff
@@ -6,9 +6,10 @@
   "standard": true,
   "userAgents": [
     {
-      "userAgent": "Opera Mobile #MAJORVER#.#MINORVER#",
+      "userAgent": "Opera Mobile #MAJORVER#.#MINORVER# Android",
```

Comment:
Could we write `Opera Mobile #MAJORVER#.#MINORVER# for Android` like in other files?

Reply:
oops, changed